### PR TITLE
Update README to include documenting changes to language and GQL

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1100.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1100.surge.sh

File: README.adoc [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
 
+This page covers the following topics:
+
+* xref:README.adoc#build-locally[]
+* xref:README.adoc#raise-prs[]
+* xref:README.adoc#documenting-changes[]
+* xref:README.adoc#cypher-gql[]
+
+
+[[build-locally]]
```
Comment (rsill-neo4j) [inline README.adoc]: not sure these have to be set
asciidoc generates anchors for headings by default - it should be `#building-locally` for the line below

edit: just noticed, on github, both anchors are present now (eg https://github.com/JPryce-Aklundh/docs-cypher/tree/updated_readme?tab=readme-ov-file#documenting-changes and https://github.com/JPryce-Aklundh/docs-cypher/tree/updated_readme?tab=readme-ov-file#documenting-changes-to-cypher)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1100.surge.sh

File: README.adoc [hunk 2]
```diff
@@ -68,3 +77,39 @@ There are a few edge cases where we might want to work only on the current branc
...
+== Documenting changes to Cypher
+
+New, updated, deprecated, and removed features must be recorded on the xref:deprecations-additions-removals-compatibility.adoc[] page.
+
+New and deprecated features should also be marked with a label:
+
+* If the impacted feature has its own header in the Cypher Manual, use the following:
+
+[role=label--new-5.x]
+\== Header
```
Comment (rsill-neo4j) [inline README.adoc]: source block missing, this doesn't render correctly:
https://github.com/JPryce-Aklundh/docs-cypher/tree/updated_readme?tab=readme-ov-file#documenting-changes-to-cypher

File: README.adoc [hunk 3]
```diff
@@ -68,3 +77,39 @@ There are a few edge cases where we might want to work only on the current branc
...
+
+[role=label--deprecated]
+\== Header
```
Comment (rsill-neo4j) [inline README.adoc]: here too

File: README.adoc [hunk 4]
```diff
@@ -68,3 +77,39 @@ There are a few edge cases where we might want to work only on the current branc
...
+
+* If the impacted feature is documented within a table (such as a return column in a `SHOW` command), use the following:
+
+`featureX` label:new[Introduced in 5.x] +
+`featureY` label:deprecated[]
```
Comment (rsill-neo4j) [inline README.adoc:103]: these work, but i guess they could use a source block as well

File: README.adoc [hunk 5]
```diff
@@ -68,3 +77,39 @@ There are a few edge cases where we might want to work only on the current branc
...
+
+Labels can be difficult to apply to updated features.
+In these cases, it is often preferable to note the change in a sentence.
+For example: "As of Neo4j 5.x, featureX supports ..."
+
+Removed features should be removed from the Cypher Manual.
```
Comment (rsill-neo4j) [inline README.adoc]: I'm a fan of imperative mood for this (usually shorter)
"remove removed ..." isn't exactly elegant though :D

File: README.adoc [hunk 6]
```diff
@@ -68,3 +77,39 @@ There are a few edge cases where we might want to work only on the current branc
...
+
+[[cypher-gql]]
+== Cypher and GQL
+
+When documenting a new Cypher feature, its relationship with GQL must be considered:
+
+* If the feature is part of GQL's mandatory features, it should be recorded on the page xref:modules/ROOT/pages/appendix/gql-conformance/supported-mandatory.adoc[].
+* If the feature is part of GQL's optional features, it should be recorded on the page xref:modules/ROOT/pages/appendix/gql-conformance/supported-optional.adoc[].
+* if the feature adds functionality for which there exists an analogous GQL equivalent, it should be recorded on the page xref:modules/ROOT/pages/appendix/gql-conformance/analogous-cypher.adoc[].
```
Comment (rsill-neo4j) [inline README.adoc]: "analogous GQL equivalent" - is it analogous or equivalent? their use might be synonymous in this context, but then we only need one :thinking:

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1100.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1100.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by rsill-neo4j — APPROVED** [review]

**Pull request merged**